workflows/sample/tasks/hifiasm_trio.wdl

```diff
@@ -103,6 +103,46 @@ task yak_trioeval {
   }
 }
 
+task yak_triobin {
+  input {
+    Int threads = 16
+    File fasta_gz
+    String yak_triobin_txt_name = "~{basename(fasta_gz)}.triobin.txt"
+    String log_name = "yak.fasta.triobin.log"
+    File parent1_yak
+    File parent2_yak
+    String pb_conda_image
+  }
+
+  Float multiplier = 2
+  Int disk_size = ceil(multiplier * size(fasta_gz, "GB")) + 20
+#  Int disk_size = 200
+  Int memory = threads * 3              #forces at least 3GB RAM/core, even if user overwrites threads
+
+  command <<<
+    echo requested disk_size =  ~{disk_size}
+    echo
+    source ~/.bashrc
+    conda activate yak
+    echo "$(conda info)"
+
+    (yak triobin  -c1 -d1 -t ~{threads} -1 ~{parent1_yak} -2 ~{parent2_yak} ~{fasta_gz} > ~{yak_triobin_txt_name} ) > ~{log_name} 2>&1
```

Comment:
```❯ yak triobin -h
Usage: yak triobin [options] <pat.yak> <mat.yak> <seq.fa>
Options:
  -c INT     min occurrence [2]
  -d INT     mid occurrence [5]
  -t INT     number of threads [8]```

Reply:
We need to remove the `-1` and `-2` from both yak calls.  This isn't part of the syntax.

```
> yak trioeval -h
Usage: yak trioeval [options] <pat.yak> <mat.yak> <seq.fa>
Options:
  -c INT     min occurrence [2]
  -d INT     mid occurrence [5]
  -n INT     min streak [2]
  -t INT     number of threads [8]

> yak triobin -h
Usage: yak triobin [options] <pat.yak> <mat.yak> <seq.fa>
Options:
  -c INT     min occurrence [2]
  -d INT     mid occurrence [5]
  -t INT     number of threads [8]
```

Reply:
Good catch. Removing.